src/author.rs

```diff
@@ -1,18 +1,40 @@
+//! The `Author` struct and related utilities.
+//!
+//! This module defines the `Author` data structure together with its utility
+//! enum `AuthorParseError`.
+
+/// The author information.
 #[derive(Debug)]
 pub struct Author {
-    name: String,
+    /// The electronic contact information of the author.
     email: String,
+
+    /// The author's name.
+    name: String,
 }
 
+/// The methods and associated functions on and for an `Author` instance,
+/// respectively.
```

Comment:
Again, this is exactly what the `impl` keyword means. We should not explain that here..